lib/index.js

```diff
@@ -34,11 +34,14 @@ module.exports = {
     }
 
     var createPackageTree = function (depTree) {
+      var targetFramework = depTree.meta ? depTree.meta.targetFramework : undefined; // TODO implement for paket and more than one framework
```

Comment:
it is called `targetRuntime` targetFramework` is .Net specific, please keep all mentions in CLI as `targetRuntime`